### PR TITLE
feat(webcentrum_eosc): new configurable service

### PR DESCRIPTION
**Pull request by Johaney-s**
* service is configurable by setting required attributes, generates general data format

### CODE REVIEW

File: send/umbraco_mu [hunk 1]
```diff
@@ -18,6 +18,7 @@ send_lib.check_destination_format(destination, destination_type)
 
 auth = send_lib.get_auth_credentials(SERVICE_NAME, destination)
 
+send_lib.create_lock(SERVICE_NAME, destination)
```
Comment (xflord) [inline send/umbraco_mu:21]: maybe this should be a separate commit (or at least a mention in the message)?

**Review by xflord — APPROVED** [review]
looks good

File: send/umbraco_mu (hunk 1 above)
Comment (Johaney-s) [reply send/umbraco_mu:21]: added to commie message
